api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/pac4j/oidc/BasePac4jOidcClientProperties.java

```diff
@@ -60,4 +60,14 @@
      * Custom parameters to send along in authZ requests, etc.
      */
     private Map<String, String> customParams = new HashMap<>();
+
+    /**
+     * Response mode.
```

Comment:
You'll need to elaborate a lot more than just simply re-stating the setting name, please.

Reply:
Also please indicate what the accepted, common values are.